src/locale/locales/es/messages.po

```diff
@@ -407,7 +407,7 @@ msgstr "Has dejado de seguir a esta cuenta"
 
 #: src/view/com/profile/ProfileMenu.tsx:98
 msgid "Account unmuted"
-msgstr "Cuenta demuteada"
+msgstr "Cuenta desmuteada"
```

Comment:
Native spanish speaker here that was going to open a PR for this so, given this one is open already...

Shouldn't we use the verb `silenciar` instead of `mute`? (Same applies for the messages on top of this one.)
https://www.rae.es/observatorio-de-palabras/mutear

Reply:
"Silenciar" is also used in this very commit so it makes sense, but I'm not sure if "desilenciar" sounds appropriate. It could be fitting to put "Has dejado de silenciar a esta cuenta", but it's a lot more verbose and the rest of the action confirmation strings would have to be changed for consistency

Reply:
for muted account should be `Cuenta Silenciada`

`Desilenciar` sound appropiate but is like a `!mute` sentence XD and not very common in spanish
so for `unmute` could be `Activar sonido" (Activate sound)

so, for unmuted accout meaby should be "Cuenta con sonido reactivado"

Reply:
or only "Cuenta con sonido"

please vote

Reply:
Like @Kloen said, we should not make any references to sound when there is none involved